testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jaxb/JaxbMarshallingSoakTest.java

```diff
@@ -80,7 +80,9 @@ public void after() throws Exception {
         Map<String, String> contextParam = new HashMap<>();
         contextParam.put("resteasy.async.job.service.enabled", "true");
         // Arquillian in the deployment use if TimeoutUtil in the deployment
-        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(new ReflectPermission("suppressAccessChecks"),
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+                new RuntimePermission("modifyThread"),
```

Comment:
Out of curiosity, is jboss-modules needing this permission?

Reply:
The call to threadPool.shutdown(); at line 162 in test is causing this permission requirement.

	at org.wildfly.security.manager.WildFlySecurityManager.checkPermission(WildFlySecurityManager.java:295)
	at org.wildfly.security.manager.WildFlySecurityManager.checkPermission(WildFlySecurityManager.java:192)
	at java.util.concurrent.ThreadPoolExecutor.checkShutdownAccess(ThreadPoolExecutor.java:735)
	at java.util.concurrent.ThreadPoolExecutor.shutdown(ThreadPoolExecutor.java:1387)
	at org.jboss.resteasy.test.providers.jaxb.JaxbMarshallingSoakTest.compare(JaxbMarshallingSoakTest.java:162)

Reply:
OK, so in this case it really makes sense to add the permission, it's testcase eventually needing that, fine.